mmv1/third_party/terraform/services/dataflow/resource_dataflow_flex_template_job_test.go.erb

```diff
@@ -1358,7 +1348,7 @@ resource "google_dataflow_flex_template_job" "flex_job_kms" {
   labels = {
    "my_labels" = "value"
   }
-  kms_key_name		= google_kms_crypto_key.crypto_key.id
+  kms_key_name		= data.google_kms_crypto_key.crypto_key.id
 
 }
 `, topicName, bucket, key_ring, crypto_key, job)
```

Comment:
This is causing the [unit test failure](https://github.com/GoogleCloudPlatform/magic-modules/actions/runs/9274836460/job/25518125320#step:8:68) (because there aren't enough formatting parameters in the string).

```suggestion
`, topicName, bucket, crypto_key, job)
```